hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -52,15 +52,22 @@ class DummyPositionJoint : public components::Joint
     if (info_.command_interfaces.size() > 1 || info_.state_interfaces.size() > 1) {
       return return_type::ERROR;
     }
+
+    hardware_interface::components::InterfaceInfo dummy_position_interface;
+    dummy_position_interface.name = HW_IF_POSITION;
+    dummy_position_interface.parameters.insert(std::make_pair<std::string, double>("min_value", -1));
+    dummy_position_interface.parameters.insert(std::make_pair<std::string, double>("max_value", 1));
+
     if (info_.command_interfaces.size() == 0) {
-      info_.command_interfaces.push_back(HW_IF_POSITION);
+      info_.command_interfaces.push_back(dummy_position_interface);
       commands_.resize(1);
     }
     if (info_.state_interfaces.size() == 0) {
-      info_.state_interfaces.push_back(HW_IF_POSITION);
+      info_.state_interfaces.push_back(dummy_position_interface);
       states_.resize(1);
     }
 
+    // TODO(andyz): Delete
```

Comment:
rather rename to `min` and `max`. And maybe use values from the `InterfaceInfo`. Although the global values are also allowed (see Example 1)